Gruntfile.js

```diff
@@ -159,7 +159,7 @@ module.exports = function (grunt) {
 
     exec: {
       start_nightWatch: {
-        command: __dirname + '/node_modules/nightwatch/bin/nightwatch' +
+        command: 'node ' + __dirname + '/node_modules/nightwatch/bin/nightwatch' +
```

Comment:
Is this a windows fix, or why do we need this change?

Reply:
Without this fix, I can't run grunt nightwatch on Windows.